### PR TITLE
Creates new jsnext bundle

### PR DESCRIPTION
**Pull request by yfr**
@developit In the tests, I had to use the default import because the named imports for Provider does not work.

### CODE REVIEW

**Comment by developit** [comment]
Ah right, I don't think I had Webpack set up for that.

**Comment by yfr** [comment]
Webpack? This repo uses rollup right? 

I would love to export the `Provider` and `connect` methods from redux as own exports. Do you know how to do that with rollup? We can just merge this if you like and i open another issue for that.

**Comment by developit** [comment]
Yeah the build is rollup, but Karma is set up to use webpack.

**Comment by yfr** [comment]
Cool, that's fine with me!

File: rollup.config.js [hunk 1]
```diff
@@ -56,7 +70,7 @@ export default {
 					modules: false
 				}]
 			].concat(babelRc.presets.slice(1)),
-			plugins: babelRc.plugins
+			plugins: ['external-helpers', ...babelRc.plugins]
```
Comment (yfr) [inline rollup.config.js]: Oh did not see that comment. Fix this first thing in the morning.

Comment (developit) [inline rollup.config.js]: Thanks!

**Comment by yfr** [comment]
@developit can you have a look?

**Comment by developit** [comment]
@yfr Looks good! Now it'll run in Node 4+ :)

**Pull request merged**